subsys/mpsl/cx/Kconfig

```diff
@@ -21,6 +21,13 @@ config MPSL_CX_PIN_FORWARDER
 
 if MPSL_CX
 
+config MPSL_CX_INIT_PRIORITY
+	int "Initialization priority of the Radio Coexistence interface"
+	default KERNEL_INIT_PRIORITY_DEFAULT
+	help
+	  Set the initialization priority number. Do not mess with it unless
+	  you know what you are doing.
```

Comment:
If there is a relationship between init priorities of MPSL_CX and its clients it would be nice it there was some build time assert to protect us from invalid configuration. We never know what the user will do for example with `CONFIG_IEEE802154_NRF5_INIT_PRIO` or what comes from upstream to us.

Reply:
Just remark for others. Trust on return code of `mpsl_cx_register_callback` is rather worthless, because before `mpsl_cx_init` which goes at the now-configurable-priority, there is "none" interface selected which simply ignores this write and it should also stay this way.

Reply:
But I'm ok to introduce this improvement later.